doc/OnlineDocs/contributed_packages/sensitivity_toolbox.rst

```diff
@@ -60,7 +60,7 @@ And finally we call sIPOPT:
     :skipif: not sipopt_available
 
     >>> m_sipopt = sipopt(m,[m.eta1,m.eta2], [m.perturbed_eta1,m.perturbed_eta2], streamSoln=True)
-    Ipopt 3.13.2: run_sens=yes
+    Ipopt ...: run_sens=yes
```

Comment:
What purpose does this have? Opening up to other versions?

Reply:
Exactly.  We have a few different versions of Ipopt floating around, so this makes the test a little more robust.